topics/ui/controls/combo_box.md

```diff
@@ -23,79 +23,86 @@ comboBox.setEditable(true);
 
 ## When to use
 
-Use a combo box if:
+Use a combo-box if:
 
-* The user needs to select a value or enter a custom value.
+* The user needs to select a value or enter a custom value:
 
-  ![](combo_box_font_size.png){width=165}
+  ![A font size combo-box that allows selecting predefined values as well as entering a custom value](combobox_when_to_use_1.png){width=706}
 
 * The possible choices are objects or states.
 
 
 * The user may need to return to previous values. For example, save previously entered paths, so the user can quickly
   select them later:
 
-  ![](maven.png){width=413}
+  ![A combo-box with an open menu with previously selected values](combobox_when_to_use_2.png){width=706}
 
 ### When not to use
 
-If the number of options is finite, and there’s no need to enter custom values. In this case, use a [drop-down list](drop_down.md) or [radio buttons](radio_button.md).
+* If the number of options is finite, and there is no need to enter custom values. In this case, use a [drop-down list](drop_down.md) or [radio buttons](radio_button.md):
```

Comment:
Same about separate sections instead of bullet points here, please see Split button as an example

Reply:
Done. Could you please check the headings?